Tests/L0/Grunt/invalidTestSource.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I believe you can merge these mock data into two files "Success/Failure" scenario. It's Ok with current mocks, however it can be fit into two simple mocks